TMDS_3/include/CGAL/Mesh_cell_base_3.h

```diff
@@ -95,14 +95,46 @@ class Mesh_cell_base_3_base<Parallel_tag>
 // Class Mesh_cell_base_3
 // Cell base class used in 3D meshing process.
 // Adds information to Cb about the cell of the input complex containing it
+/*!
+\ingroup PkgMesh3MeshClasses
+<!-- Meta-comment: this class cannot be deprecated by
+Compact_mesh_cell_base_3, because the later has a different API.
+-- Laurent Rineau, 2013/10/16
+\deprecated This class is deprecated since \cgal 4.3. Use
+`CGAL::Compact_mesh_cell_base_3<GT,MD,Tds>` instead.
+-->
+
+The class `Mesh_cell_base_3<GT, MD, Cb>` is a model of the concept `MeshCellBase_3`.
+It is designed to serve as cell base class for the 3D triangulation
+used in the 3D mesh generation process.
```

Comment:
The doc may be in Mesh_3, but this is still code that you read while opening a file from TMDS_3...

Reply:
right. What do you think should be changed here?